packages/backend/server/src/models/doc.ts

```diff
@@ -461,4 +468,82 @@ export class DocModel extends BaseModel {
   }
 
   // #endregion
+
+  // #region DocMeta
+
+  /**
+   * Create or update the doc meta.
+   */
+  async upsertMeta(
+    workspaceId: string,
+    docId: string,
+    data?: DocMetaUpsertInput
+  ) {
+    return await this.db.workspaceDoc.upsert({
+      where: {
+        workspaceId_docId: {
+          workspaceId,
+          docId,
+        },
+      },
+      update: {
+        ...data,
+      },
+      create: {
+        ...data,
+        workspaceId,
+        docId,
+      },
+    });
+  }
+
+  /**
+   * Get the doc meta.
+   * @param options.isPublic: if true, only return the public doc. If false, only return the private doc.
+   * If not set, return public or private both.
+   */
+  async getMeta<Select extends Prisma.WorkspaceDocSelect>(
```

Comment:
why is public filter reuqired?

Reply:
I searched and found that the code is not being used, I removed it